guide/introduction.md

```diff
@@ -1,5 +1,36 @@
-# Membrane Framework
+# Membrane Framework guide
+Hello and welcome to the general guide to the Membrane Framework!
 
-Hello and welcome to the general guide to the Membrane Framework! If you see something requiring improvement in this guide, feel free to create an issue or open a PR in [this repository](https://github.com/membraneframework/guide). For more information about the framework and more tutorials, check our website: [membrane.stream](https://membrane.stream).
+Membrane is a multimedia processing framework that focuses on reliability, concurrency, and scalability. It is primarily written in Elixir, while some platform-specific or time-constrained parts are written in C. With a range of existing packages and an easy-to-use interface for writing your own, Membrane can be used to process almost any type of multimedia.
```

Comment:
```suggestion
Membrane is a multimedia processing framework that focuses on reliability, concurrency, and scalability. It is primarily written in Elixir, while some platform-specific or time-constrained parts written in C. With a range of existing packages and an easy-to-use interface for writing your own, Membrane can be used to process almost any type of multimedia.
```

Reply:
I think that `are` is crucial: "while some ... parts ARE written in C". Alternatively we can think of writing: "with some ... parts written in C"